src/shaders/heatmap.fragment.glsl

```diff
@@ -0,0 +1,21 @@
+#pragma mapbox: define highp float weight
+
+uniform float u_intensity;
+varying vec2 v_extrude;
+
+// Gaussian kernel coefficient: 1 / sqrt(2 * PI)
+#define GAUSS_COEF 0.3989422804014327
+
+void main() {
```

Comment:
Suggest renaming this program from `heatmap` to `heatmap_density`, and then renaming `heatmap_texture` to just `heatmap`

Reply:
Not sure those names would be clearer. Also, I named them similarly to the extrusion names — `fill_extrusion` and `extrusion_texture`.